changelog.d/5375.wip

```diff
@@ -0,0 +1 @@
+Dynamically showing/hiding onboarding personalisation screens based on the the users homeserver capabilities
```

Comment:
```suggestion
Dynamically showing/hiding onboarding personalisation screens based on the users homeserver capabilities
```

Reply:
great catch! sometimes you see the words so often they disappear...